### ISSUE TITLE
Option value "type" validation

### ISSUE THREAD

**Issue by crookse** [drashland/line#20]
If we decide that options should take in multiple values, then we should think about this as well.

## Summary

What:

Currently, the option validation for option values only checks to see that the option value was specified:

 - If the value is missing, then an error shows up.

We should do option value validation on the "type" of the option value. For example, I have this option:

```
public options = {
  "--connections [value: number]": "Add the number of connections"
};
```

With the above option, Line should check that the value is a number when it parses the option.

Why:

It would be nice to have built-in option value type validation.

## Acceptance Criteria

Below is a list of tasks that must be completed before this issue can be closed.

- [ ] Write documentation
- [ ] Write unit tests
- [ ] Write integration tests

### Help Menu

```text
Test Creator - Make test files

USAGE

    test-creator [option]
    test-creator [options] [arg: test_filename] [arg: num_test_cases]

OPTIONS

    -h, --help
        Show this menu.
    -v, --version
        Show this CLI's version.

    --connections <number>
        Add the number of connections
```
